### ISSUE TITLE
Downsampling sum-type flows data: like vlm

### ISSUE THREAD

**Issue by goodboy** [pikers/piker#326]
As per #302 we now aggressively leverage the [m4 graphics downsampling algo](www.vldb.org/pvldb/vol7/p797-jugel.pdf) for redic low latency chartingn on massive data sets, however this algo *reduces* based on maintaining *range data* on screen; this is not desired for *accumulation* type sampled data (such as in volume or histogram/bar charts) where we want to simply aggregate (aka `sum()`) all datums from an uppx range into each horizontal pixel slot.

In other words we need to adjust our implementation to accept additional [reduction function(s) besides `max()`/`min()`](https://github.com/pikers/piker/blob/ab135d2b7578f163d66c1af4c0c7d3ead8936aa2/piker/ui/_compression.py#L306) so that when you zoom out on such data sets you see the **accumulation** of the data in each set of uppx-spanning pixel-bins instead of a lossy range representation as is used for price data.
